extensions/amp-story-interactive/0.1/interactive-disclaimer.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LocalizedStringId} from '../../../src/localized-strings';
+import {htmlFor} from '../../../src/static-template';
+
+/**
+ * Disclaimers will retrieve the information from the lookup dictionary below.
+ * In order to add a "Learn more" link or entity name ("Your response will be sent to <Organization>"),
+ * submit a PR with a new entry on the BACKENDS dictionary and tag @wg-stories to review it.
+ */
+
+/** @const {!Object<string, !Object>} */
+const BACKENDS = {
+  'webstoriesinteractivity-beta.web.app': {
+    learnMoreUrl: 'https://policies.google.com/terms',
+    entityName: 'Google Firebase',
+  },
+};
+
+/**
+ * Creates a disclaimer icon and dialog.
+ * @param {!Element} element
+ * @return {!Element}
+ */
+function buildDisclaimerLayout(element) {
+  const html = htmlFor(element);
+  return html`<div class="i-amphtml-story-interactive-disclaimer">
+    <button
+      class="i-amphtml-story-interactive-disclaimer-alert"
+      aria-label="Open disclaimer"
+    >
+      <div class="i-amphtml-story-interactive-disclaimer-alert-icon"></div>
```

Comment:
This could be a pseudo element of  `.i-amphtml-story-interactive-disclaimer-alert` since it's decorative.

Then add  `content: '';` and  `display: block; to the styling`
I think you could also remove `pointer-events: none` if you do that too.